src/component/header/header/Search.jsx

```diff
@@ -1,15 +1,32 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { getArticles } from "../../../api";
 
-const Search = ({ allArticles, setAllArticles }) => {
+const Search = ({ setOrder, setSortBy }) => {
   const [inputSearchValue, setInputSearchValue] = useState("");
+  const [chosenOrder, setChosenOrder] = useState("asc");
+  //
+  const [chosenSortBy, setChossenSortBy] = useState("");
+  //
+
   const pressedToSearch = () => {
-    getArticles(inputSearchValue).then(({ articles }) =>
-      setAllArticles(articles)
-    );
+    getArticles(inputSearchValue).then(({ articles }) => setOrder(articles));
   };
+
+  useEffect(() => {
+    setOrder(chosenOrder);
+    setSortBy(chosenSortBy);
+  }, [chosenOrder, chosenSortBy, setOrder, setSortBy]);
   return (
     <div id="search">
+      <select onClick={(e) => setChosenOrder(e.target.value)}>
+        <option value={"asc"}>Ascend</option>
+        <option value={"desc"}>Descen</option>
+      </select>
+      <select onClick={(e) => setChossenSortBy(e.target.value)}>
+        <option value={""}>Date</option>
+        <option value={"votes"}>Votes</option>
+        <option value={"article_id"}>Article ID</option>
+      </select>
       <input onChange={(e) => setInputSearchValue(e.target.value)}></input>
       <button onClick={pressedToSearch}>Seaech</button>
     </div>
```

Comment:
This search bar does not currently function - I'd recommend giving the user limited options for how they can search, rather than an open input field. The articles displayed are not currently being updated when different options are selected from the dropdown.